### ISSUE TITLE
UI improvement for saving new sessions

### ISSUE THREAD

**Issue by Clindbergh** [ReDEnergy/SessionSync#59]
Thanks for this great addon and development!

When saving a new session explicitely (by clicking the heart icon) I would specifically name that session. This currently requires 5 Clicks:

1. Session Sync Toolbar Icon
2. Heart icon to save the session
3. Right click on the newly added session to edit its name
![grafik](https://user-images.githubusercontent.com/2742027/51090208-a697ee80-1778-11e9-9c3a-19a96a1e0028.png)
4. Click Edit
5. Highlight the current session name in the edit field with mouse click-dragging (or CTRL + A) 
![grafik](https://user-images.githubusercontent.com/2742027/51090304-47d37480-177a-11e9-92d2-d8f325e9e6e9.png)

6. Save the session with a click on the save button (or ENTER)

It would be nice if the we could skip steps 3-5 (including). So the new flow would be:

1. Click on Session Sync Toolbar Icon
2. Heart Icon to save session
3. The edit text field for changing is presented with current text highlighted. After typing text, click the save button (or ENTER)
![grafik](https://user-images.githubusercontent.com/2742027/51090289-10fd5e80-177a-11e9-992d-a2bc7383528a.png)

**Comment by ReDEnergy** [comment]
Hi. I don't get what's happening.
What you described in second section is the actual behaviour that is suppose to be happening.
Is it not working in your case ?
I mean... right after you on the click the save icon.. the edit box with the session name should appear.

**Comment by ReDEnergy** [comment]
Oh... forgot.
The thing is... if you have more than 1 windows saved. I can't show that box. The edit box will only be triggered if you save only one window, because otherwise I can't know for sure which one should be edited and even if I would select one of them it might be a little strange to only change one.

P.S you can also double-click on the session entry to rename it. No need to go through the context menu.

**Comment by ReDEnergy** [comment]
Oh... actually I think I found a bug. Thanks for reporting it.
Seems that what you described only works if the **Multi-Window** toggle is on, and you have just one window active. 
![image](https://user-images.githubusercontent.com/1395510/51090435-8cfca400-1784-11e9-9812-7d35749d856e.png)

This is a stupid bug, no idea how I missed it when testing. I will try to provide a bug-fix as soon as possible.

**Comment by ReDEnergy** [comment]
I fixed the issue. Also added auto-select. I will update the new version in a few days just so that I can test is a little bit to make sure I didn't brake anything (I fixed a few other issues as well so better safe than sorry)

If you want to test it you can download the [signed addon from here](https://www.dropbox.com/s/zobr1q4b464huie/session_sync-3.1.2beta-fx.xpi?dl=0). It's exactly as it's suppose to be delivered in a few days, but has 3 debug buttons in the addon menu.

**Comment by Clindbergh** [comment]
Do you plan to push the code for the new signed addon to the repository? I noticed that the last commit was in July 2018.

**Comment by ReDEnergy** [comment]
Yes, I'll do that once I push the addon to oficial release. I kinda messed up my local repository because I had working on several complex features and I need to do some cleanup first to merge them properly. Probably I will do this today or tomorrow
